discord-scripts/thread-management/check-thread-archiving.ts

```diff
@@ -246,14 +256,14 @@ async function updateThreadStatusFromMessage(
   const { channel: thread, createdTimestamp: messageTimestamp } = message
   if (
     !thread.isThread() ||
-    isInRecreationalCategory(thread) ||
-    !isInPermittedCategoryOrChannel(robot.brain, thread, "archive-checking") ||
+    // isInRecreationalCategory(thread) ||
+    // !isInPermittedCategoryOrChannel(robot.brain, thread, "archive-checking") ||
```

Comment:
Ohp, why'd we comment these out?

Reply:
Brought back `isInRecreationalCategory` but left `!isInPermittedCategoryOrChannel` since that check is causing some issues against testing channels. We can implement that as the main call once we remove `!isInTestingChannel(thread)`